src/stylesheets/main.scss

```diff
@@ -344,3 +344,9 @@ p code {
     margin-bottom: 0;
   }
 }
+
+// Add styling for 'what's new' banner
+.app-whats-new {
+  border: 1px solid $govuk-border-colour;
```

Comment:
This is more of a design decision but I wonder if the border should only be at the top and the bottom? Having it go around the container looks a bit off especially with inverse colours since the other containers are limited width. 
![Screenshot 2021-06-24 at 16 19 41](https://user-images.githubusercontent.com/5007934/123290051-dc745c00-d508-11eb-973c-e0f1259fe6e3.png)

Reply:
Yep, I'll change that. thanks Hanna.

Reply:
If I made the top border transparent would it still be visible on high contrast mode? @vanitabarrett I thought you might be able to help me :)

Reply:
This is what it looks like with the top and bottom border in Windows High Contrast Mode

<img width="1168" alt="Screenshot 2021-06-25 at 10 37 09" src="https://user-images.githubusercontent.com/503614/123405201-7daef100-d5a1-11eb-972e-5cff6ceed010.png">

Reply:
Yeah I'm not sure the top border looks very good in non High Contrast Mode. I'm trying to work out if there's an approach where you can see it in High Contrast but that looks 'invisible' without.
<img width="857" alt="Screenshot 2021-06-25 at 10 42 03" src="https://user-images.githubusercontent.com/56260216/123405761-0cbc0900-d5a2-11eb-99a2-eee9f7b2177f.png">

Reply:
There is an upcoming CSS media query that will let you style things specifically for inverted colours: https://developer.mozilla.org/en-US/docs/Web/CSS/@media/inverted-colors
 
 It's only supported in Safari so far though :(

Reply:
There's also `forced-colors`, which is similar, and works for Windows High Contrast Mode, but not Safari 🤦 

https://developer.mozilla.org/en-US/docs/Web/CSS/@media/forced-colors

Reply:
Adding a border to the bottom of the masthead actually does the trick, with no need for media queries:

```diff
diff --git a/src/stylesheets/components/_masthead.scss b/src/stylesheets/components/_masthead.scss
index bce5929c..d8202989 100644
--- a/src/stylesheets/components/_masthead.scss
+++ b/src/stylesheets/components/_masthead.scss
@@ -3,6 +3,7 @@
   @include govuk-responsive-padding(6, "bottom");
   color: govuk-colour("white");
   background-color: govuk-colour("blue");
+  border-bottom: 1px solid govuk-colour("blue");
 }
 
 .app-masthead__title {
diff --git a/src/stylesheets/main.scss b/src/stylesheets/main.scss
index 0487bd8f..35e25fb5 100644
--- a/src/stylesheets/main.scss
+++ b/src/stylesheets/main.scss
@@ -347,7 +347,6 @@ p code {
 
 // Add styling for 'what's new' banner
 .app-whats-new {
-  border-top: 1px solid $govuk-border-colour;
   border-bottom: 1px solid $govuk-border-colour;
   background-color: $app-light-grey;
 }
```

![Screenshot 2021-06-25 at 12 03 03](https://user-images.githubusercontent.com/503614/123415535-5100d680-d5ad-11eb-8bc9-bbeacaafea5a.png)